mergify_engine/context.py

```diff
@@ -752,6 +753,66 @@ async def retrieve_unverified_commits(self) -> typing.List[str]:
             if not commit["commit_verification_verified"]
         ]
 
+    async def retrieve_resolved_review_threads(
+        self, resolved: typing.Optional[bool] = True
+    ) -> ContextAttributeType:
```

Comment:
The returns type is too broad, it should be just what return the function `typing.List[...]`